lib/active_model/entity/schemas/json.rb

```diff
@@ -49,8 +49,14 @@ def make_schema_nullable!(options)
             options[:nullable] = true if options[:$ref]
           end
 
+          def append_description_if_available!(name, options)
+            key = name.underscore.to_sym
+            options[:description] = meta_descriptions[key] if meta_descriptions.key?(key)
```

Comment:
```suggestion
            options[:description] = meta_descriptions[key].presence
```

Reply:
if `description: null` is not fit  -- ignore

Reply:
ignored